addons/mail/static/src/core/common/composer_model.js

```diff
@@ -94,6 +94,10 @@ export class Composer extends Record {
     get syncHtmlWithMessage() {
         return this.message && !this.isDirty;
     }
+
+    get targetThread() {
+        return this.replyToMessage?.thread ?? this.thread ?? null;
```

Comment:
do we really need the fallback on null?

Reply:
I don't think so, this is a move from component to record for the sake of avoiding a `owner.thread`.
To make sure to not break things in last minute, let's keep it there for now.